examples/problem_decomposition/mifno.ipynb

```diff
@@ -404,11 +376,7 @@
     "\n",
     "In order to assess the impact of quantum workflows applied to one or several fragments on the total energy of the system, the helper class provides a `mi_summation` method. This class recomputes the total energy of the system, using the reference values obtained classically by QEMIST Cloud, or instead using the values obtained by your quantum workflows for the fragments you specify. This summation was described by the formulas in the first section of this notebook.\n",
     "\n",
-    "If no argument is passed to `mi_summation`, then only reference values are used in the summation, which should then be in agreement (up to machine precision) with `fno_fragments.e_tot`, originally read from the QEMIST Cloud result blob.\n",
-    "\n",
-    "Otherwise, passing a dictionary using fragment labels as keys and the energy as the corresponding values does the trick. For these fragments, MP2 correction is automatically added, and a new total energy is computed.\n",
-    "\n",
-    "The difference between this new total energy and the reference total energy gives you a measure of the accuracy and the impact of combining quantum workflows with classical calculations performed with MI-FNO."
+    "If no argument is passed to `mi_summation`, then only reference values are used in the summation, which should then be in agreement (up to machine precision) with `fno_fragments.e_tot`, originally read from the QEMIST Cloud result blob. Otherwise, passing a dictionary using fragment labels as keys and the energy as the corresponding values does the trick. For these fragments, MP2 correction is automatically added, and a new total energy is computed. The difference between this new total energy and the reference total energy gives you a measure of the accuracy and the impact of combining quantum workflows with classical calculations performed with MI-FNO."
```

Comment:
the end of this statement is kind of not super compatible with the eigenvalue diagonalization in the sense than no quantum algorithm was run. Also, isnt the MP2 correction only valid for MI FNO and not for iFCI (or at least we should say how it differs in what can be observed by the user)